include/mqt-core/circuit_optimizer/CircuitOptimizer.hpp

```diff
@@ -90,7 +90,8 @@ class CircuitOptimizer {
    * @param qc the quantum circuit
    * @param maxBlockSize the maximum size of a block
    */
-  static void collectBlocks(QuantumComputation& qc, std::size_t maxBlockSize);
+  static void collectBlocks(QuantumComputation& qc, std::size_t maxBlockSize,
+                            bool collectCliffords);
```

Comment:
```suggestion
                            bool onlyCollectCliffords = false);
```
Maybe that name would be a little more suitable?
Also, this parameter is missing a description in the corresponding docstring.
Finally, this should default to false, otherwise this would be a breaking change.